sherpa/__init__.py

```diff
@@ -116,11 +116,25 @@ def test(level=1, verbosity=1, datadir=None):
 
 
 def clitest():
-    try:
-        import pytest, pytest_cov
-        errno = pytest.main(['--pyargs', 'sherpa', '-rs'])
-        sys.exit(errno)
-    except ImportError:
-        print """Cannot import pytest and pytest-cov.
-            Please run 'pip install -r test_requirements.txt' first"""
-        sys.exit(1)
+    def install(package_name):
+        try:
+            import pip
+            pip.main(['install', package_name])
+        except:
+            print """Cannot import pip or install packages with it.
+            You need pytest, and possibly pytest-cov, in order to run the tests.
+            If you downloaded the source code, please run 'pip install -r test_requirements.txt' first.
+            """
```

Comment:
Should it say "please run 'pip install ...' from the source directory first." ?

I don't know when `clitest` can be called - I assume it is called by `sherpa_test` - and I just feel it's worth being explicit about the location of the `test_requirements.txt` file. If `clitest` can only ever be (sensibly) called from the directory containing `setup.py` (and hence `test_requirements.txt`) then it's not really an issue.